Sources/Swift-Big-Number-Core.swift

```diff
@@ -1257,6 +1280,11 @@ fileprivate extension Array where Element == Limb
 
 	/// Returns the number of bits that contribute to the represented number, ignoring all
 	/// leading zeros.
+	
+	// FIXME: - Not the correct definition of *bitWidth* according to Apple - MG 6 Feb 2022
```

Comment:
Please add a unit test to match the correct definition

Reply:
Can do.